pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -409,26 +401,57 @@ def compile(
                 replay_from=replay_from,
             )
         install_for_distribution()
+    del bash_runner.env["_PYTHON_HOST_PLATFORM"]
+
+
+def package_wheel(pkg_name, pkg_root, srcpath, build_metadata, bash_runner):
+    """
+    Parameters
```

Comment:
Nit: docstrings need a 1 line description followed by an empty line )

```suggestion
    """Package a wheel
    
    Parameters
```